src/core/mac/mac_frame.hpp

```diff
@@ -560,6 +592,25 @@ class Frame : public otRadioFrame
      */
     void SetSrcAddr(const Address &aAddress);
 
+    /**
+     * This method gets the Security Control Field.
+     *
+     * @param[out]  aSecurityControlField  The Security Control Field.
+     *
+     * @retval OT_ERROR_NONE   Successfully retrieved the Security Level Identifier.
+     * @retval OT_ERROR_PARSE  Fail to find the security control field ins the frame.
+     *
+     */
+    otError GetSecurityControlField(uint8_t &aSecurityControlField) const;
+
+    /**
+     * This method sets the Security Control Field.
+     *
+     * @param[in]  aSecurityControlField  The Security Control Field.
+     *
+     */
+    void SetSecurityControlField(uint8_t aSecurityControlField);
```

Comment:
Is it possible to use bool?

Reply:
This is **Security Control Field** in **Auxiliary Security Header**. It consists of eight bits, as an example:
```
Security Control Field: 0x0d
    .... .101 = Security Level: Encryption with 32-bit Message Integrity Code (0x5)
    ...0 1... = Key Identifier Mode: Indexed Key using the Default Key Source (0x1)
    ..0. .... = Frame Counter Suppression: False
    .0.. .... = ASN in Nonce: False
    0... .... = Reserved: 0x0
```
Do you mean the **Security Enabled** bit in Frame Control Field?